src/microbit/model/image.py

```diff
@@ -0,0 +1,289 @@
+# implementing image model as described here:
+# https://microbit-micropython.readthedocs.io/en/latest/image.html

```

Comment:
Noticed the placement of this is different from Button class. We should probably keep it consistent.